protocol-dashboard/src/components/TransactionStatus/TransactionStatus.tsx

```diff
@@ -78,7 +78,7 @@ const WaitingTransaction: React.FC<WaitingTransactionProps> = props => {
     setError,
     setStatus,
     cancelTransaction
-  } = useCancelTransaction(props.name)
+  } = useCancelTransaction(props.name, !isOpen)
```

Comment:
would prefer if this hook returned a fn `reset` that's called in this component in a `useEffect`, to make the behavior more predictable

but if you want to go this route could you change the params be named params (instead of positional)? client + money decided that if there are 2+ params they should be named

Reply:
This is the same pattern used across most of the hooks (10+) in the protocol dashboard for fetching data. I'll make a ticket for that work, totally agree that the return reset pattern is way better/safer. But want to be careful to not touch too many things here and I think having consistency at least for now is good.

https://linear.app/audius/issue/C-3831/protocol-dashboard-hooks-should-return-a-resetclean-up-fn-instead-of